package.json

```diff
@@ -149,10 +149,10 @@
     "@types/webpack-merge": "^4.1.3",
     "@types/winston": "^2.2.0",
     "@types/xml2js": "^0.4.0",
-    "electron": "1.8.7",
-    "electron-builder": "20.2.0",
+    "electron": "2.0.4",
+    "electron-builder": "20.19.2",
```

Comment:
Just tested this upgrade out in a Linux VM (we only use this for generating Linux installers) and I can still build, package and launch Desktop from there :tada:

<img width="1021" src="https://user-images.githubusercontent.com/359239/42592434-ddfd0b76-851f-11e8-8f0d-be16ac609a6c.png">